### ISSUE TITLE
Missing logs for nats-server after migration to v2

### ISSUE THREAD

**Issue by b1tamara** [cloudfoundry/nats-release#66]
### Observed behavior:

After successful migration from gnats to nats-server v2 the logs from NATS Server are missing. It is impossible to debug any issues with NATS Servers now. Only logs from nats-migrate-server and bmp are still here.

During looking at the code, we faced out 
- bpm starts [nats-wrapper](https://github.com/corlettb/nats-release/blob/c764f492b52f33a9fd7be5583629dba46f9a6b46/jobs/nats-tls/templates/bpm.yml.erb#L19)
- nats-wrapper checks the bin_path, which is in our case `/var/vcap/packages/nats-server/bin/nats-server`
- nats-wrapper starts the nats-server v2 via [command.Start ](https://github.com/corlettb/nats-release/blob/c764f492b52f33a9fd7be5583629dba46f9a6b46/src/code.cloudfoundry.org/nats-v2-migrate/nats-wrapper/main.go#L183) without forwarding the logs from the command to stdout /stderr. 

### Expected behavior:
The NATS Server logs regarding client / routes connection / disconnection or slow consumer info should be available
E.g.
`[6] 2020/09/28 13:01:59.102430 [INF] 10.1.1.X:4225 - rid:868 - Route connection created`

### Current version: 
nats-release: 56.8.0

nats config:
```
net: "<...>"
port: 4224
prof_port: 0
http: "0.0.0.0:8223"
write_deadline: "2s"

debug: false
trace: false
logtime: true


tls {
  ca_file: "/var/vcap/jobs/nats-tls/config/external_tls/ca.pem"
  cert_file: "/var/vcap/jobs/nats-tls/config/external_tls/certificate.pem"
  key_file: "/var/vcap/jobs/nats-tls/config/external_tls/private_key.pem"
  cipher_suites: [
    "TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384",
    "TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256"
  ]
  curve_preferences: [
    "CurveP384"
  ]
  timeout: 5 # seconds
  verify: true
}

cluster {
  no_advertise: true
  host: "<...>"
  port: 4225



  tls {
    ca_file: "/var/vcap/jobs/nats-tls/config/internal_tls/ca.pem"
    cert_file: "/var/vcap/jobs/nats-tls/config/internal_tls/certificate.pem"
    key_file: "/var/vcap/jobs/nats-tls/config/internal_tls/private_key.pem"
    cipher_suites: [
      "TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384",
      "TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256"
    ]
    curve_preferences: [
      "CurveP384"
    ]
    timeout: 5 # seconds
    verify: true
  }


  routes = [

    nats-route://<...>.nats.service.cf.internal:4225

    nats-route://<...>.nats.service.cf.internal:4225

    nats-route://<...>.nats.service.cf.internal:4225

  ]
}
```

**Comment by b1tamara** [comment]
Are there any plans to get a rid of nats-wrapper (incl. nats-migration-server) and to start nats-server directly by bpm?

**Comment by b1tamara** [comment]
@reneighbor 
@mariash

**Comment by b1tamara** [comment]
slack thread about it: https://cloudfoundry.slack.com/archives/C01ABMVNE9E/p1698915133075279

**Comment by domdom82** [comment]
Fixed in #71
